benchmarks/power_distribution/power_distributor.py

```diff
@@ -47,10 +51,12 @@ async def send_requests(batteries: set[int], request_num: int) -> list[Result]:
         List of the results from the PowerDistributingActor.
     """
     battery_pool = microgrid.battery_pool(batteries)
-    results_rx = battery_pool.power_distribution_results()
-    result: list[Result] = []
+    results_rx = battery_pool.power_status.new_receiver()
+    result: list[Any] = []
     for _ in range(request_num):
-        await battery_pool.set_power(Power(float(random.randrange(100000, 1000000))))
+        await battery_pool.propose_power(
+            Power(float(random.randrange(100000, 1000000)))
```

Comment:
Optional (and specially because this PR is not really adding these numbers): this will soon fail when we incorporate [`flake8-numbers`](https://github.com/frequenz-floss/flake8-numbers) (and I think is a perfect example of where it adds clarity :D):
```suggestion
            Power(float(random.randrange(100_000, 1_000_000)))
```

Reply:
It is not new, can be fixed in flake8 PR.